modules/7_cluster_annotation/cluster_annotation.qmd

```diff
@@ -71,6 +71,13 @@ library(ggplot2)
 library(future)
 library(patchwork)
 
+library(clusterProfiler)
```

Comment:
Only load libraries that you cannot refer to during function calls, e.g. `openxlsx::read.xlsx`.

Reply:
I have kept this as such for know because clusterprofiler sometimes give strange errors. Will do some testing and then adapt this